src/window.rs

```diff
@@ -94,7 +96,7 @@ impl<'a> ToGlibPtr<'a, *mut gdk_sys::GdkWindowAttr> for WindowAttr {
 
         let mut attrs = Box::new(gdk_sys::GdkWindowAttr {
             title: title.0 as *mut c_char,
-            event_mask: self.event_mask,
+            event_mask: i32::try_from(self.event_mask.bits()).unwrap(),
```

Comment:
This would fail if the top bit is set. Should just be a cast

Reply:
fixed.